### ISSUE TITLE
Learning rate choice question

### ISSUE THREAD

**Issue by aldakata** [allenai/OLMo-core#373]
I just read the technical report. Thank you very much for the wealth of knowledge that you are open-sourcing!

I have a question, in page 17, right before section 4 you say (see appended screenshot):
"the high learning rate setting is 2.8 points better than the lower learning rate. More study is needed to turn this interesting data point into a dependable result. This finding contradicts machine learning folk wisdoms such as “higher learning rates are always better” or “area under the learning curve matters”".

You say that the **high learning rate** is better and that **this finding contradicts ml folk wisdomws such as "higher learning rates are always better"**. 
You ultimately choose the smaller learning rate, did I read this right? Is there a typo somewhere?

<img width="1230" height="239" alt="Image" src="https://github.com/user-attachments/assets/bb56d7fe-e657-4382-853c-2f7099425d52" />

Thank you very much for your time,

I really appreciate your work

**Comment by dirkgr** [comment]
I see how this is confusing, especially since those two paragraphs ended up right next to each other.

The way to read this is this: Across all learning rate settings, annealing recipes, and metrics that we used, there was no difference between the high and the low learning rates, overturning folk wisdom.

The one exception that we found was on math, with the GSM8K evaluation set, in the setting where we switch to datasets to be more math-heavy. You might conclude that high learning rates still confer an advantage when shifting distributions, but we did not do enough experiments to conclude that firmly.

**Comment by aldakata** [comment]
Beautiful, thank you very much!
May I add a question:
In figure 11, page 16, you show a small study of different LRs. 
I am wondering whether you have the checkpoints of this experiment saved somewhere and whether I could have them?

I am not comfortable openly sharing my project (soon to be on arxiv), but I am happy to share what I have found privately, and what I would do with the checkpoints. I am sure you would find it interesting!

Thank you again for your time

<img width="981" height="383" alt="Image" src="https://github.com/user-attachments/assets/7ac749be-9d26-408e-8939-fd44a7de99c6" />

**Comment by dirkgr** [comment]
I can send these to you, but they take up quite a lot of space. About 200TB or so. Email me and we'll work out how exactly to do it?

**Comment by rozeappletree** [comment]
Basically 0.2 petabyte, crazy.

**Comment by dirkgr** [comment]
I should close this out! We did this, and @aldakata shared some very cool results with me. @aldakata , will they be published soon? Do you want to link here?

**Comment by aldakata** [comment]
Yes! I will be presenting them at https://efficient-reasoning.github.io/ workshop at NeurIPS next week. 🔌 Pre-print https://arxiv.org/abs/2510.06213

If anyone is curious, we studied the impact of pretraining choices on quantization error. OLMo2's LR exploration (as seen in the above Figure 11) allowed us to verify whether our findings would hold at scale. They do 🥳,

<img width="2528" height="1048" alt="Figure 23" src="https://github.com/user-attachments/assets/bd8767d6-bd9e-4749-b500-48f6f2ca5ec4" />

P.S.: Thanks again for the checkpoints! And congratulations for OLMo3
